pkg/controllers/accesslogpolicy_controller.go

```diff
@@ -265,12 +273,12 @@ func (r *accessLogPolicyReconciler) buildAndDeployModel(
 	if err != nil {
 		return nil, err
 	}
-	r.log.Debugw("Successfully built model", "stack", jsonStack)
+	r.log.Debugw(context.TODO(), "Successfully built model", "stack", jsonStack)
```

Comment:
for log lines that currently do not have a context in the scope, I added a todo for now